Classes/common/CDTReplicator/CDTReplicator.h

```diff
@@ -16,6 +16,7 @@
 #import <Foundation/Foundation.h>
 
 #import "CDTReplicatorDelegate.h"
+#import "NSURLSessionConfigurationDelegate.h"
```

Comment:
Not being an expert in building as frameworks for iOS, I want to clarify if this is necessary or just a bit of confusion caused by my naming of `NSURLSessionConfigurationDelegate.h`.

`NSURLSessionConfigurationDelegate.h` is a file I created, not part of the iOS foundation framework. I can understand the confusion though as I chose to name it in such a way that it ended up sounding like it's part of the foundation framework. I chose the name because it's a delegate for setting up the `NSURLSessionConfiguration`, so it made sense (to me) from that perspective, but it is a little confusing.